core/app/services/spree/import_service/products/update.rb

```
@@ -0,0 +1,62 @@
+          Spree::ImportService::Products::Serializers::OptionTypesAttributesSerializer.new(row: row, product_id: product_id).to_a.tap do |attr|
+            Spree::ProductOptionType.insert_all(attr)
+          end
+
+          Spree::ImportService::Products::Serializers::OptionValuesAttributesSerializer.new(row: row, variant_id: variant.id).to_a.tap do |attr|
+            Spree::OptionValueVariant.insert_all(attr)
+          end
+        end
+      end
+    end
+  end
+end
```

RuboCop (1.75.5)

[convention] Final newline missing.

(Layout/TrailingEmptyLines)